app/controllers/courses/vacancies_controller.rb

```diff
@@ -61,6 +63,7 @@ def update_vacancies_for_multiple_sites
                                    end
           site_status.save
         end
+      SiteStatus.send_vacancies_filled_notification(course_id: @course.id) if multiple_sites_no_vacancies?
```

Comment:
I've written up a ticket here https://trello.com/c/3KNjTG6A/620-add-validation-to-vacancies-page
We'll get this pr merged now to avoid the current bug, but validation needs to be added for this!